src/arguments/default/cls-expr-async-gen-meth-static.template

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
```

Comment:
These are the same templates used in the src/function-forms/default folder (old src/params/default).

I'm copying them.